docs/gop_sql_engine.md

```diff
@@ -0,0 +1,41 @@
+# 论go的mysql依赖库哪家合适
```

Comment:
please use https://github.com/goplus/community/blob/main/docs/mysql-framework.md instead of creating new file.

Reply:
And please use English as first language.